entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -86,6 +88,12 @@ class EntityService(
             throw BadRequestDataException("Entity ${expandedEntity.id} targets unknown entities: $inErrorRelationships")
         }
 
+        if (!expandedEntity.propertiesHaveAtMostOneDefaultInstance())

```

Comment:
Maybe not in this jira but I think it's time to put all of this validation in a separate component

Reply:
Jira 😨 (please, no).

PS: open an issue for this validation topic.

Reply:
btw, there is already some validation set up in ExpandedEntity, it might be a better place.

Reply:
what bothers me here is that we don't say to the user which property has the error